src/data/mob-name.csv

```diff
@@ -290,7 +290,7 @@
 "アングータ","Anguta"
 "アイタンリ","Eitanli"
 "ジュエルジルコン","Jewel Zircon"
-"ビロンギングス","Billongings"
+"ビロンギングス","Bellongings"
```

Comment:
not Belongings?

Reply:
Hmmm.... yeah, Belongings sounds better.

Reply:
```suggestion
"ビロンギングス","Belongings"
```